meson.build

```diff
@@ -0,0 +1,79 @@
+project(
+    'minmea',
+    'c',
+    default_options: ['c_std=c99', 'buildtype=debug'],
+)
+
+add_project_arguments(
+    '-Wall', '-Wextra', '-Werror',
+    '-D_POSIX_C_SOURCE=199309L', '-D_BSD_SOURCE', '-D_DEFAULT_SOURCE',
+    '-D_DARWIN_C_SOURCE',
+    language: 'c',
+)
+
+if host_machine.system() == 'windows'
+    add_project_arguments('DMINMEA_INCLUDE_COMPAT')
+endif
+
+c_args = []
+
+if get_option('buildtype') == 'debug'
+    c_args += '-ggdb'
```

Comment:
The debug buildtype already sets -g, why do you need to enforce -ggdb (which the Makefile doesn't do anyway)?